### PR TITLE
removed references to authUtils from the controllers

### PR DESCRIPTION
**Pull request by shriramshankar**
This is first of the many PRs to use the user information added to the request body and request headers instead of decoding the token everytime the user information is needed.

### CODE REVIEW

**Comment by shriramshankar** [comment]
@iamigo @pallavi2209 my subsequent PRs are based on this. Can you review this one?

File: api/v1/controllers/userTokens.js [hunk 1]
```diff
@@ -88,32 +88,17 @@ module.exports = {
...
-
-      // Check if requesting user is Admin
-      return authUtils.isAdmin(req);
-    })
-    .then((isAdmin) => {
-      // if admin, delete the token
-      if (isAdmin) {
-        return tokenToDel.destroy();
+      /*
+       * Ok to destory the token if the request is made by an admin user or
```
Comment (iamigo) [inline api/v1/controllers/userTokens.js]: destroy

**Comment by iamigo** [comment]
disallowMultipleLineBreaks: ; and describe should have at most 2 line(s) between them at tests/api/v1/verbs/doPost.js :
    12 |'use strict';
    13 |const expect = require('chai').expect;
    14 |const pu = require('../../../../api/v1/helpers/verbs/postUtils');
-------------------------------------------------------------------------^
    15 |
    16 |
1 code style error found.

**Comment by shriramshankar** [comment]
@iamigo @pallavi2209 fixed the lint errors and updated the PR.

**Pull request merged**